v1/download/download.go

```diff
@@ -231,13 +226,18 @@ func (d *Downloader) loop(ctx context.Context) {
 			return
 		}
 
+		// Note: MaxDelaySeconds, MinDelaySeconds and LongPollingTimeoutSeconds
+		// are scaled from int seconds to ns in ValidateAndInjectDefaults.
```

Comment:
This majorly caught me out while working on this, so leaving here for my future self!